bots/running-a-bbot-bot/README.md

```diff
@@ -0,0 +1,184 @@
+```
+
+Run the docker image, passing environment configs:
+
+```
+docker run -it --rm --name brocket \
+  -e ROCKETCHAT_URL=<YOUR_URL> \
+  -e ROCKETCHAT_USER=<BOT_USER> \
+  -e ROCKETCHAT_PASSWORD=<BOT_PASS> \
+  -e ROCKETCHAT_ROOM='' \
+  -e LISTEN_ON_ALL_PUBLIC=true \
+  -e RESPOND_TO_EDITED=true \
+  -e RESPOND_TO_DM=true \
+  -e RESPOND_TO_LIVECHAT=true \
+  bbot:rocketchat
+```
+
+## Deploy to Heroku
+
+Heroku can detect the Node app type and automatically run the main script.
+
+You should install the [Heroku CLI][heroku-cli] and login. Either create your
+Heroku app following the linked instructions or add an existing app to your
+bot's git as a remote host, per instructions below.
+
+It just needs the environment configuration:
+
+```
+heroku config:set ROCKETCHAT_URL=<YOUR_URL>
+heroku config:set ROCKETCHAT_USER=<BOT_USER>
+heroku config:set ROCKETCHAT_PASSWORD=<BOT_PASS>
+heroku config:set ROCKETCHAT_ROOM=''
+heroku config:set LISTEN_ON_ALL_PUBLIC=true
+heroku config:set RESPOND_TO_DM=true
+heroku config:set RESPOND_TO_EDITED=true
+heroku config:set RESPOND_TO_LIVECHAT=true
+```
+
+Then add your Heroku app as a git remote and push to release:
+
+```
+heroku git:remote -a <YOUR_APP>
+git push heroku master
+```
+
+## Deploy Docker to Heroku
+
+Log in to Container Registry
+
+```
+heroku container:login
+```
+
+Build the image and push to Container Registry:
+
+```
+heroku container:push web
+```
+
+Then release the image to your app:
+
+```
+heroku container:release web
+```
+
+Now open the app in your browser:
+
+```
+heroku open
+```
+
+## Deploy to Glitch
+
+Remix the [Glitch boilerplate][brocket-glitch].
+
+Edit **.env** as per example:
+
+```
+ROCKETCHAT_URL=<YOUR_URL>
+ROCKETCHAT_USER=<BOT_USER>
+ROCKETCHAT_PASSWORD=<BOT_PASS>
+RESPOND_TO_DM=true
+RESPOND_TO_EDITED=true
+LISTEN_ON_ALL_PUBLIC=false
+```
+
+That's it, the app will restart and attempt connection when you change the
+settings, just view the logs to resolve any connection issues.
+
+Glitch apps will usually sleep after 5 minutes, but
+[there are ways](support.glitch.com/t/not-letting-the-app-go-down/2000/13)
+to get round that for periods of time.
+
+## Deploy to Now
+
+[Now from Zeit.co](https://zeit.co/now) provide free global serverless
+deployments, with up to 1GB of bandwidth and storage.
+
+First create an account, then install the Now CLI.
+
+Deploy your app, passing environment configs:
+
+```
+now -e ROCKETCHAT_URL=<YOUR_URL> \
+  -e ROCKETCHAT_USER=<BOT_USER> \
+  -e ROCKETCHAT_PASSWORD=<BOT_PASS> \
+  -e ROCKETCHAT_ROOM='' \
+  -e LISTEN_ON_ALL_PUBLIC=true \
+  -e RESPOND_TO_EDITED=true \
+  -e RESPOND_TO_DM=true \
+  -e RESPOND_TO_LIVECHAT=true
+```
+
+Choose `package.json` or `Dockerfile` as the build source.
+
+Follow the link (in clipboard) to your newly created app to view logs.
+
+## Contribute to the Docs
+
+If you've developed deployment methods other common cloud platforms, please
+submit a PR with the instructions using the link below.
```

Comment:
I don't see any links below, maybe is missing?

Reply:
Every page has a "Contribute to this page on GitHub" link appended, that's all it's referring to. I was just adding a preface to call out that button and prompt for specific contributions.